dvc/command/stage.py

```diff
@@ -34,7 +34,7 @@ def is_plot_or_metric(out: "BaseOutput"):
 
     outs = list(filterfalse(is_plot_or_metric, stage.outs))
     if outs:
-        desc.append("Produces " + part_desc(outs))
+        desc.append("Outputs " + part_desc(outs))
```

Comment:
@jorgeorpinel, still not sure about `metrics/params` part, so I am leaving `Generates` part as-is. 

For others, here's the discussion around it: https://github.com/iterative/dvc/pull/5282#discussion_r560641053

Reply:
I don't see why "Generates" is any more specific to metrics or plots than "Outputs." Maybe "Reports" would be a bit more specific, but I'm not sure it's worth it to have a different verb. That being said, this is truly nitpicking and seems unlikely to even be noticed by users, so I'm fine to leave as is if this is your preference @skshetry .

Reply:
I think `Reports` is what I am looking for, though I find it to be a bit _formal_. Thanks @dberenbaum.